enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/RestAPI.java

```diff
@@ -156,12 +160,26 @@ public String init(Api api) {
                 .enableBackendJWT(api.getEnableBackendJWT()).backendJWTConfiguration(backendJWTConfiguration)
                 .deploymentType(api.getDeploymentType())
                 .environmentId(api.getEnvironmentId())
-                .environmentName(api.getEnvironmentName()).build();
+                .environmentName(api.getEnvironmentName())
+                .choreoComponentInfo(choreoComponentInfo)
+                .build();
 
         initFilters();
         return basePath;
     }
 
+    private static org.wso2.choreo.connect.enforcer.commons.model.ChoreoComponentInfo getChoreoComponentInfo(
```

Comment:
same here

Reply:
a class with the same name already imported. Hence had to improve with the fully qualified class name.